SDK v2 migration guide.md

```diff
@@ -138,6 +138,11 @@ Find a client you currently use below, read the table of API name changes and us
   - The supported workaround is to provide a client web socket instance in the client options.
 - The authentication classes for devices and modules have been consolidated.
 - Reduced access levels to classes and methods that were never intended to be public where possible.
+- The device and module clients now only support `IAsyncDisposable`, which will ensure `CloseAsync()` is called before disposing.
+  - The syntax for this might be quite new for some and feel awkward. You can choose from the following options:
+    1. Manually call `await client.DisposeAsync();`.
+    1. Initialize the client with the **await** keyword: `await using client = new IotHubDeviceClient(...);`. The client will be disposed when it goes out of scope.
```

Comment:
In markdown, you can label each numbered bullet with 1 and it will autoincrement for you.

<https://www.markdownguide.org/basic-syntax/#ordered-lists>

Reply:
Nope, with IAsyncDisposable, you use both `await using` as a prefix to your disposable type.

Reply:
Ah, I misunderstood the second BP. Thanks!